Model/Config/Source/Mode.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Copyright © PureClarity. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+namespace Pureclarity\Core\Model\Config\Source;
+
+use Magento\Framework\Data\OptionSourceInterface;
+
+/**
+ * Class Mode
+ *
+ * contains list of valid modes for use in config
+ */
+class Mode implements OptionSourceInterface
+{
+    /** @var string Client Side Mode - Default */
+    const MODE_CLIENTSIDE = 'clientside';
+
+    /** @var string Server Side Mode */
+    const MODE_SERVERSIDE = 'serverside';
+
+    /**
+     * Default PureClarity valid mode labels
+     *
+     * @var string[]
+     */
+    private $validModes = [
+        self::MODE_CLIENTSIDE => 'Client-side',
```

Comment:
Client-side (line 30)   vs clientside (line 19)?

Reply:
"clientside" is what would be stored in the db, "Client-side" is the label displayed to the user.